### DISCUSSION TITLE
What about fediverse?

### DISCUSSION THREAD

**Discussion by noriban** [stoatchat/discussions#605] [General]
Hi,

I think revolt is great for fediverse, thanks to this, a lot of users from the fediverse network can come, integration with other services (mastodon). What do you think?

**Comment by Rexogamer** [comment]
Revolt is not federated and we don't *currently* have any plans to add support for federation - you can read more about our reasons [here](https://developers.revolt.chat/faq/federation). 

In addition, if we *were* to add any kind of federation we'd probably either limit it to federation between Revolt instances or with other chat platforms - I don't quite see how Mastodon and Revolt would work very well together 🤔

**Reply by benjamin051000** [reply]
Yeah I think the idea isn't to connect it to mastodon lol. But rather to connect revolt instances together.
